readme.txt

```diff
@@ -82,7 +82,6 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
 - Dev: Add `woocommerce_admin_export_id` filter for customizing the export file name #7178
 - Dev: Remove old payment gateway task components #7224
 - Fix: Currency display on Orders activity card on homescreen #7181
-- Fix: Fix obsolete key property in gateway defaults #7229
```

Comment:
Was this accidentally deleted?

Reply:
@joshuatf  Thank you for the review! I rebased the changelog.